docs/kyma/docs/036-inst-custom-component-installation.md

```diff
@@ -0,0 +1,67 @@
+
+## Remove a component
+
+>**NOTE:** Not all components can be simply removed from the Kyma installation. In case of Istio and the Service Catalog, you must provide your own deployment of these components in the Kyma-supported version before you remove them from the  installation process. See [this](https://github.com/kyma-project/kyma/blob/master/resources/istio-kyma-patch/templates/job.yaml#L25) and [this](https://github.com/kyma-project/kyma/blob/master/resources/service-catalog/charts/catalog/values.yaml#L3) documents accordingly to check the currently supported versions of Istio and the Service Catalog.
+
+To disable a component from the list of components that install with Kyma, remove this component's entries from the appropriate file. The file depends on whether you install Kyma from the release or from sources, and whether you install Kyma locally or on a cluster. The version of your component's deployment must match the version that Kyma currently supports.
+
+### Installation from the release
+
+1. Download the [newest version](https://github.com/kyma-project/kyma/releases) of Kyma.
+2. Customize installation by removing a component from the list of components in the Installation resource. For example, to disable the Application Connector installation, remove this entry:
+    ```
+    name: "application-connector"
+    namespace: "kyma-system"
+    ```
+  * from the `kyma-config-local.yaml` file for **local** installation
+  * from the `kyma-config-cluster.yaml` file for the **cluster** installation
+
+
+3. Follow the installation steps described in the [Install Kyma locally from the release](#installation-install-kyma-locally-from-the-release) document, or [Install Kyma on a GKE cluster](#installation-install-kyma-on-a-gke-cluster) accordingly.
+
+### Installation from sources
+
+1. Customize installation by removing a component from the list of components in the following Installation resource:
+  * [installer-cr.yaml.tpl](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr.yaml.tpl)  for the **local** installation
+  *  [installer-cr-cluster.yaml.tpl](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster.yaml.tpl) for the **cluster** installation
+
+2. Follow the installation steps described in the [Install Kyma locally from sources](#installation-install-kyma-locally-from-sources) document, or [Install Kyma on a GKE cluster](#installation-install-kyma-on-a-gke-cluster) accordingly.
+
+### Verify the installation
+
+1. Check if all Pods are running in the `kyma-system` Namespace:
+  ```
+  kubectl get pods -n kyma-system
+  ```
+2. Sign in to the Kyma Console using the `admin@kyma.cx` as described in the [Install Kyma locally from the release](#installation-install-kyma-locally-from-the-release) document.
```

Comment:
Shouldn't the referenced doc names be in bold? - that comment applies to all references in this PR.

Reply:
Since we already provide links, I do not think we should put doc names in bold. It was a temporary solution invented for cases where linking was not possible.